core/src/main/scala/org/apache/gearpump/cluster/AppDescription.scala

```diff
@@ -18,11 +18,11 @@
 
 package org.apache.gearpump.cluster
 
-import scala.reflect.ClassTag
+import java.io._
```

Comment:
please avoid star imports. You may configure the Intellij "auto start imports" count to a large number (e.g. 100).